### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by singhravipratap**
### Reasons for including paper:

- Paper gives attention mechanism also known as Transformers architecture.
-
-

### CODE REVIEW

**Comment by DarrenN** [comment]
👋🏽 Thanks for the PR.

It looks like there's a small issue with the formatting, in that you have the `*` inside a `>` blockquote, which incorrectly indents the entry.

Also, "Attention" is misspelled in the link.

**Comment by zeeshanlakhani** [comment]
@singhravipratap please fix when possible, and we'll get this in.

**Comment by singhravipratap** [comment]
@DarrenN, @zeeshanlakhani, Thanks for the review, Updated the record.

File: artificial_intelligence/README.md [hunk 1]
```diff
@@ -12,4 +12,6 @@
...
\ No newline at end of file
+    > This paper proposes a method for translating music across musical instruments, genres, and styles. It is based on a multi-domain wavenet autoencoder, with a shared encoder and a disentangled latent space that is trained end-to-end on waveforms. Employing a diverse training dataset and large net capacity, the domain-independent encoder enables translation even from musical domains that were not seen during training. The method is unsupervised and does not rely on supervision in the form of matched samples between domains or musical transcriptions. This method is evaluated on NSynth, as well as on a dataset collected from professional musicians, and achieve convincing translations, even when translating from whistling, potentially enabling the creation of instrumental music by untrained humans.
+
+* [Attention is all you need](http://papers.neurips.cc/paper/7181-attention-is-all-you-need.pdf)) by Ashish Vaswani et al.
```
Comment (zeeshanlakhani) [inline artificial_intelligence/README.md:17]: @singhravipratap 

super minor, but there's an extra `)`. Can you remove it?

File: artificial_intelligence/README.md [hunk 2]
```diff
@@ -12,4 +12,6 @@
...
\ No newline at end of file
+    > This paper proposes a method for translating music across musical instruments, genres, and styles. It is based on a multi-domain wavenet autoencoder, with a shared encoder and a disentangled latent space that is trained end-to-end on waveforms. Employing a diverse training dataset and large net capacity, the domain-independent encoder enables translation even from musical domains that were not seen during training. The method is unsupervised and does not rely on supervision in the form of matched samples between domains or musical transcriptions. This method is evaluated on NSynth, as well as on a dataset collected from professional musicians, and achieve convincing translations, even when translating from whistling, potentially enabling the creation of instrumental music by untrained humans.
```
Comment (zeeshanlakhani) [inline artificial_intelligence/README.md:15]: Does this need the extra space?

**Review by zeeshanlakhani — CHANGES_REQUESTED** [review]

**Comment by TaiTaiTai777** [comment]
When evaluating a paper of this nature, we should consider several aspects including clarity, novelty, methodology, and experimental validation. Here's a comprehensive review:

### Clarity (Rating: ★★★☆☆)
- **Strengths:** The paper's abstract is fairly clear in describing what the proposed method does and its underlying technology. The use of terms like "multi-domain wavenet autoencoder," "shared encoder," and "disentangled latent space" indicates a solid, technical base.
- **Weaknesses:** The abstract contains several typographical errors and some fragmented notation (like "mus ic" instead of "music"). This can detract from readability and professional presentation.

### Novelty (Rating: ★★★★★)
- **Strengths:** The concept of translating music across instruments, genres, and styles without supervision is highly innovative. The use of a multi-domain wavenet autoencoder that can generalize to unseen musical domains adds to its novelty. The idea that even whistling can be translated into instrumental music is particularly intriguing and original.
- **Weaknesses:** None apparent from the abstract.

### Methodology (Rating: ★★★★☆)
- **Strengths:** The use of a shared encoder and disentangled latent space in a multi-domain wavenet autoencoder is described as the core of the method. Training it end-to-end on waveforms and using a diverse dataset underscores a rigorous approach. The unsupervised nature of the method is a significant advantage as it removes the need for expensive and time-consuming labeled data.
- **Weaknesses:** Specific details on the implementation, training procedure, and architecture are missing from the abstract, though this is expected in a brief summary. 

### Experimental Validation (Rating: ★★★★☆)
- **Strengths:** Evaluation on both the NSynth dataset and a custom dataset from professional musicians demonstrates a broad testing ground. The statement that the method achieves "convincing translations" provides confidence in its practical applications.
- **Weaknesses:** Quantitative metrics and comparison with existing methods are not provided in the abstract. This would be necessary in the full paper to substantiate the claims.

### Overall Rating: ★★★★☆ (4/5)

#### Summary:
This paper presents a promising, innovative method for translating music across various domains using a multi-domain wavenet autoencoder. The approach is unsupervised and generalizes well to unseen domains, including non-instrumental sounds like whistling. However, the abstract would benefit from a proofread to correct typographical errors and further elaboration on the experimental results and methodology details in the full paper. Despite these minor issues, the work stands out for its novelty and potential impact.

File: artificial_intelligence/README.md [hunk 3]
```diff
@@ -12,4 +12,6 @@
 * [Mastering the Game of Go with Deep Neural Networks and Tree Search](http://airesearch.com/wp-content/uploads/2016/01/deepmind-mastering-go.pdf) by Silver et al.
```
Comment (Gon128) [inline artificial_intelligence/README.md:12]: 414545455

Comment (Gon128) [reply artificial_intelligence/README.md:12]: C

**Review by NaNOteh — APPROVED** [review]

File: artificial_intelligence/README.md (hunk 2 above)
Comment (erictleung) [reply artificial_intelligence/README.md:15]: The diff is hard to see, but this paragraph previously has a trailing space, so this commit actually strips off the extra whitespace, which I think is fine.

File: artificial_intelligence/README.md (hunk 1 above)
Comment (erictleung) [inline artificial_intelligence/README.md:17]:
```suggestion
* [Attention is all you need](http://papers.neurips.cc/paper/7181-attention-is-all-you-need.pdf) by Ashish Vaswani et al.
```

Comment (NaNOteh) [reply artificial_intelligence/README.md]: > ```suggestion
> 
> * [Attention is all you need](http://papers.neurips.cc/paper/7181-attention-is-all-you-need.pdf) by Ashish Vaswani et al.
> 
> ```

File: artificial_intelligence/README.md [hunk 4]
```diff
@@ -12,4 +12,6 @@
...
 
 * [A Universal Music Translation Network (2018)](https://arxiv.org/pdf/1805.07848.pdf) by Noam Mor, Lior Wolf, Adam Polyak & Yaniv Taigman
```
Comment (NaNOteh) [inline artificial_intelligence/README.md:14]: Sarong

Comment (NaNOteh) [reply artificial_intelligence/README.md:14]: Sarong

**Pull request merged**